bravado/mapping/docstring.py

```diff
@@ -82,58 +121,63 @@ def create_operation_docstring(operation):
         :type from_date: str
         :rtype: list
     """
+    print 'creating op docstring for %s' % operation.operation_id
     s = ""
-    op_dict = operation.operation_dict
-    is_deprecated = op_dict.get('deprecated', False)
+    op_spec = operation.operation_spec
+    is_deprecated = op_spec.get('deprecated', False)
     if is_deprecated:
         s += "** DEPRECATED **\n"
 
-    summary = op_dict.get('summary')
+    summary = op_spec.get('summary')
     if summary:
         s += "[{0}] {1}\n\n".format(operation.http_method.upper(), summary)
 
-    desc = op_dict.get('description')
+    desc = op_spec.get('description')
     if desc:
         s += "{0}\n\n".format(desc)
 
-    for param_dict in op_dict.get('parameters', []):
-        s += create_param_docstring(param_dict)
+    for param_spec in op_spec.get('parameters', []):
+        s += create_param_docstring(param_spec)
 
-    responses = op_dict.get('responses')
-    for http_status_code, response_dict in iter(sorted(responses.iteritems())):
-        response_desc = response_dict.get('description')
+    responses = op_spec.get('responses')
```

Comment:
It could lead to responses being `None` and later `None.iteritems()` throwing up. Though, `responses` should atleast contain one element, i would still add a default value `{}` along.